languages/python/oso/polar/errors.py

```diff
@@ -92,10 +89,6 @@ def _operational_error(subkind, message, details):
         return OperationalError(message, details)
 
 
-def _roles_validation_error(message, details):
```

Comment:
Do we need to remove these errors from java?

Reply:
I don't see them in Java